library/SimplePie/File.php

```diff
@@ -148,7 +143,7 @@ public function __construct($url, $timeout = 10, $redirects = 5, $headers = null
 							$this->redirects++;
 							$location = SimplePie_Misc::absolutize_url($this->headers['location'], $url);
 							$previousStatusCode = $this->status_code;
-							$this->__construct($location, $timeout, $redirects, $headers, $useragent, $force_fsockopen);
```

Comment:
For configurations using the manual redirect method, the cURL options were broken.